public/index.html

```diff
@@ -70,6 +70,16 @@ <h1 class="title-block__title">Popular destinations</h1>
         class="fa-solid fa-chevron-left chevron chevron--left"></i><i
         class="fa-solid fa-chevron-right chevron chevron--right"></i>
     </div>
+    <header class="title-block title-block__promo-card container">
+      <h2 class="title-block__title">Special offers</h2>
+      <div class="title-block__separator"></div>
```

Comment:
Hi! plis just think abput  a better naming for "separator" xD

Reply:
Thanks! Will change ASAP😅